packages/app/components/login/index.web.tsx

```diff
@@ -77,6 +80,16 @@ export function Login({ onLogin }: LoginProps) {
               : `Pushed a request to your wallet...`}
           </Text>
         </View>
+      ) : showSignMessage ? (
+        <View tw="py-20 px-10">
+          <Text tw="text-center text-lg dark:text-gray-400">
+            We need a signature in order to verify your identity. This won't
+            cost any gas.
+          </Text>
+          <Button tw="mt-8" onPress={verifySignature}>
+            Sign a message
```

Comment:
Maybe we could say `Sign the message` instead?

Reply:
yeah, makes sense.